src/test/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurerTests.java

```diff
@@ -125,7 +127,13 @@ public void testUpdateAdminPasswordWithWeakPassword() throws NoSuchFieldExceptio
             System.setSecurityManager(null);
         }
 
-        verifyStdOutContainsString("Password weakpassword is weak. Please re-try with a stronger password.");
+        verifyStdOutContainsString(
+                String.format(
+                        "Password weakpassword failed validation: \"%s\". Please re-try with a minimum %d character password that is strong per zxcvbn validation.",
+                        INVALID_PASSWORD_INVALID_REGEX.message(),
```

Comment:
Should I include more tests for each of the other validation failures? (e.g. too short password, weak password)

Reply:
You can add 1 more test around password length validation, otherwise the test-cases should suffice.

Reply:
Thanks! Added a "too short" test case in revision 2.